<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion Club</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="studentTab">
       <attribute name="title">
        <string>Students</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="studentSearchEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="studentSearchButton">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="studentTableView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="addStudentButton">
            <property name="text">
             <string>Add Student</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editStudentButton">
            <property name="text">
             <string>Edit Student</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteStudentButton">
            <property name="text">
             <string>Delete Student</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="clubTab">
       <attribute name="title">
        <string>Clubs</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="clubSearchEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="clubSearchButton">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="clubTableView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="addClubButton">
            <property name="text">
             <string>Add Club</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editClubButton">
            <property name="text">
             <string>Edit Club</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteClubButton">
            <property name="text">
             <string>Delete Club</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="membershipTab">
       <attribute name="title">
        <string>Memberships</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLineEdit" name="studentFilterEdit">
            <property name="placeholderText">
             <string>Filter by student...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="clubFilterEdit">
            <property name="placeholderText">
             <string>Filter by club...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="membershipTableView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="addMembershipButton">
            <property name="text">
             <string>Add Membership</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeMembershipButton">
            <property name="text">
             <string>Remove Membership</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
